.travis.yml

```diff
@@ -1,25 +1,45 @@
+dist: xenial
+
+os: linux
```

Comment:
these are defaults, it will just silence some infos

![image](https://user-images.githubusercontent.com/556268/118841585-bc79c900-b8c8-11eb-8ca5-6004a70cef47.png)